UPGRADE-3.2.md

```diff
@@ -6,6 +6,7 @@ FrameworkBundle
 
  * The `Controller::getUser()` method has been deprecated and will be removed in
    Symfony 4.0; typehint the security user object in the action instead.
+ * The `Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\AddConsoleCommandPass` has been deprecated. Use `Symfony\Component\Console\DependencyInjection\AddConsoleCommandPass` instead.
```

Comment:
Please also add this to the `UPGRADE-4.0.md` (but reword it to "removed" there).